internal/storage/inmemory.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+type Storage struct {
+	URLMap map[string]string
+}
+
+func New() *Storage {
+	return &Storage{
+		URLMap: make(map[string]string),
```

Comment:
тип данных уже означает что это map - дублировать в названии не имеет смыла - просто URLs
```suggestion
		URLs: make(map[string]string),
```